README.md

```diff
@@ -4,11 +4,15 @@
 [![coveralls][coveralls-badge]][coveralls-url]
 [![snyk][snyk-badge]][snyk-url]
 
-Udaru is a policy based authorization module that can be used to add permissions to 'actions' and 
-'resources'. Udaru supports 'organizations', 'teams', and 'users'; policies can be created for 
-each. Udaru can be used as a stand-alone module , or as a stand-alone server or 
+Udaru is a policy based authorization module that can be used to add permissions to 'actions' and
+'resources'. Udaru supports 'organizations', 'teams', and 'users'; policies can be created for
+each. Udaru can be used as a stand-alone module , or as a stand-alone server or
```

Comment:
it's not true for organization (yet). The db structure is there, but there is no api endpoint to add policies to an organization